backend/models/conn.js

```diff
@@ -1,12 +1,14 @@
 const pgp = require('pg-promise') ({
     query: function(e) {
-        console.log('query query')
+        console.log('query', e.query)
     }
 });
 
 const options = {
     host: "localhost",
-    database: "crm"
+    database: "crm", 
+    user: 'alazar',
+    password: 'password'
 };
 
 const db = pgp(options);
```

Comment:
you can add a gitignore in your directory for conn.js so GIT does not push up your DB username and password every time for easier merging.